compiler/env/OMRCompilerEnv.cpp

```diff
@@ -118,6 +120,7 @@ OMR::CompilerEnv::initializeTargetEnvironment()
 
    // Initialize the target CPU by querying the host processor
    //
+   target.cpu = TR::CPU::detect(TR::Compiler->omrPortLib);
```

Comment:
currently there isn't an option to set the target processor. We can easily add something to achieve that though.
For portable AOT there will be a "relocatableTarget.cpu" just like target. And for every compilation, it will be initialized be either "target" or "relocatableTarget" depending on whether it's an AOT or JIT compilation.

Reply:
ok. Even for "relocatableTarget.cpu" we need a way to specify which processor we want to target. Some people may want a newer processor because they can sure the AOT code is always going to run at least on that processor level.